tests/integration-e2e/test_coreneuron_directmode.py

```diff
@@ -24,7 +34,9 @@ def test_coreneuron_no_write_model(USECASE3):
                                                                 4.2, 5.5, 7., 7.4, 8.6]))
     npt.assert_allclose(spike_dict["node_ids"][:10], np.array([0, 1, 2, 0, 1, 2, 0, 0, 1, 2]))
 
-    soma_reader = ElementReportReader(SimConfig.reports.get("soma_report").get('FileName'))
+    soma_file = SimConfig.reports.get("soma_report").get("FileName")
```

Comment:
Assuming `SimConfig.reports` is a dict, since we don't want the `None` return from `.get`, does it make sense to use `SimConfig.reports["soma_report"]["FileName"]`?